contracts/dtoken/src/borrow.rs

```diff
@@ -2,6 +2,77 @@ use crate::*;
 
 #[near_bindgen]
 impl Contract {
+    pub fn borrow(&mut self, token_amount: WBalance)  -> Promise {
```

Comment:
@dariacherniak `pendatic mode on` 
extra space between bracket and arrow `)  ->`
`pendatic mode off`